README.md

```diff
@@ -51,4 +45,58 @@ jobs:
         run: echo "How many linked issues? ${{ steps.check-linked-issues.outputs.linked_issues_count }}"
 ```
 
+ When the job fails, it will add a new comment by default. You can also write your custom comment:
```

Comment:
we need to highlight that this is another, different example from the one above, otherwise it's confusing for the user. the first example is the standard case, this second one is if you want to use this extra feature. also note that in your second example, the comment option should be set to false because it's supposed to be a boolean option